src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs

```diff
@@ -116,5 +122,38 @@ public static IApplicationBuilder UseWebJobsScriptHost(this IApplicationBuilder
 
             return builder;
         }
+
+        private static void UseHealthChecks(IApplicationBuilder app)
```

Comment:
You might consider turning this into an extension method on IApplicationBuilder, so you can use it more cleanly like builder.UseHealthChecks().